### DISCUSSION TITLE
Question : Commands to operate Maps and other features

### DISCUSSION THREAD

**Discussion by rptango** [johnkeel-thork/DMD2#87] [Q&A] [answered]
HI, I assume this is a recurrent question you may receive.
I'm working on a personal project to create my own controller (based on arduino) and so far I have not found what shortcuts does the DMD2 Map app use for Zoom In/Out and Center the Map.
do you have any documentation or insight you can share me on what shortcuts are available on DMD2 ? (example I would love to have a shortcut to go directly to map or to go thru the Different DMD2 Modes ).

thanks in advance!,
Roy

**Comment by johnkeel-thork** [comment, marked as answer]
Sorry for the reply!
We haven't been using our public GitHub that much but we will be reviving it soon.
As for controllers we only support CARPE Controller and block any other for the reason the sales of the controllers is our main financing source for development.
We do hope to be able to do more in the future related to supporting other controllers.
